cmd/kubeadm/app/componentconfigs/kubelet_windows.go

```diff
@@ -69,8 +71,6 @@ func mutatePaths(cfg *kubeletconfig.KubeletConfiguration, drive string) {
 	}
 
 	// Mutate the fields we care about.
-	klog.V(2).Infof("[componentconfig] kubelet/Windows: changing field \"resolverConfig\" to empty")
-	cfg.ResolverConfig = ptr.To("")
```

Comment:
this can be a problem on windows and i think we should keep it.
the kubelet default value is not "" and windows must have "" as any other value will cause an error.

Reply:
I see, kubelet does not use "" as the default value, but `/etc/resolv.conf`.